backend/app/Repositories/EloquentChartVisitorsRepository.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Repositories;
+
+use App\Actions\Visitors\GetNewChartVisitorsByDateRangeRequest;
+use App\Repositories\Contracts\ChartVisitorsRepository;
+use Carbon\Carbon;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\DB;
+
+class EloquentChartVisitorsRepository implements ChartVisitorsRepository
+{
+    public function getNewVisitorsByDate(GetNewChartVisitorsByDateRangeRequest $request): Collection
+    {
+        $userId = Auth::user()->id;
+        $startData = Carbon::createFromTimestampUTC($request->getStartDate())->toDateTimeString();
+        $endData = Carbon::createFromTimestampUTC($request->getEndDate())->toDateTimeString();
```

Comment:
$userId, $startDate, $endDate pass as parameters, and calculate them inside action